nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/layers/convolution/Upsampling2d.java

```diff
@@ -55,6 +57,28 @@ public Upsampling2d(SameDiff sameDiff, SDVariable input, boolean nchw, int scale
 
     public Upsampling2d() {}
 
+    public Upsampling2d(@NonNull INDArray input,@NonNull int scale) {
```

Comment:
`@NonNull` annotation doesn't make sense on a primitive.
int, double, float, etc are Java primitive types, not objects, hence can't be null by definition.
There's another instance of this below.